client/js/entities/model/health.js

```diff
@@ -0,0 +1,17 @@
+define(function(){
+
+return function Health(life) {
+	var self = this;
+
+	self.life = life || 100;
+
+	self.clone = function() {
+		return new Health(self.life);
+	};
+
+	self.lerp = function(other, frac) {
+		self.life = self.life*(1 - frac) + other.life*frac;
+	};
```

Comment:
You may say no, but I want to add more to "Health". Like, health of particular limbs perhaps? Stuff like that. We'll see.